pycbc/detector.py

```diff
@@ -408,7 +438,7 @@ def project_wave(self, hp, hc, ra, dec, polarization,
             import lalsimulation
             h_lal = lalsimulation.SimDetectorStrainREAL8TimeSeries(
                     hp.astype(np.float64).lal(), hc.astype(np.float64).lal(),
-                    ra, dec, polarization, self.frDetector)
+                    ra, dec, polarization, self.lal())
```

Comment:
It feels like we would want to cache `self.lal()` so it does not get recomputed all the time??